tests/test_pe.py

```diff
@@ -6,6 +6,7 @@
 import sys, random, unittest
 import string as st
 from binascii import unhexlify
+import os
```

Comment:
import os can join in to the upper part import. alphabetical order of coz

Reply:
Nothing in the first section is alphabetically ordered?
Where do you want the os to go exactly?

Reply:
import os, random, sys, unittest

should be like that.

Reply:
Thanks.
I tried to follow the code around it, without making any unrelated changes, hence why I did not consider reordering them all :)